app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/TimeUtils.scala

```diff
@@ -130,128 +130,6 @@ object TimeUtils {
 
     }
 
-  def dateFormatterForMultiDisposals(
```

Comment:
integrate this `dateFormatterForMultiDisposals` function logic into `dateFormatter` and fix compilation issues and tests